src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -65,26 +72,60 @@
     private final transient String key;
 
     /**
+     * Request for fetching app token.
+     */
+    private final transient com.jcabi.http.Request trequest;
+
+    /**
+     * Request for verifying user credentials.
+     */
+    private final transient com.jcabi.http.Request vcrequest;
```

Comment:
@adko-pl ok let's keep both requests but we should rename them to avoid composite names. how about `token` and `user` ?